### DISCUSSION TITLE
Is there a way to specify a specific branch or commit for a Fastlane plugin?

### DISCUSSION THREAD

**Discussion by simondelphia** [fastlane/fastlane#20437] [Q&A] [answered]
I am trying out the [xchtmlreport plugin](https://github.com/TitouanVanBelle/fastlane-plugin-xchtmlreport) but the latest release tag doesn't include the latest changes on its main branch. Is there a way to tell Fastlane to install a plugin from the latest commit of a specific branch or a specific commit instead of based on the latest release tag?

**Comment by simondelphia** [comment, marked as answer]
I guess I can just do something like this (in the Pluginfile)
```
gem "fastlane-plugin-[plugin_name]", git: "the_git_url", branch: "main"
```

Seems the `git` parameter is necessary in order to use the `branch` parameter.

**Reply by joshdholtz** [reply]
Glad you found your answer! I also just made a PR for this on docs to make this hopefully more clear - https://github.com/fastlane/docs/pull/1167
